modules/client-configuration/pages/registration-methods-saltbundle.adoc

```diff
@@ -0,0 +1,33 @@
+[[registration-methods-saltbundle]]
+= Client Registration with Salt Bundle
+
+The registration method with the Salt bundle is the recommended registration method as long as it does not cover your use case.
```

Comment:
Could it be that the "not" should not be in the second half of this sentence?

Reply:
The meaning of this sentance is not clear.

Reply:
Yes, sth went wrong.  Now I write (submitted locally):

> The registration method with the Salt bundle is the recommended registration method as long as it covers your use case.